### PR TITLE
Homework4

### PR DESCRIPTION
**Pull request by Mazepin**
Hi. Its test Homework, not for check :)

### CODE REVIEW

File: src/Mazepin.Homework4/Program.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace Mazepin.Homework4
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            IZooManager zooManager = new IZooManager();
```
Comment (diman84) [inline src/Mazepin.Homework4/Program.cs]: this is not going to compile )

**Review by diman84 — COMMENTED** [review]
Please show some examples of inheritance and polymorphism in the application

File: src/Mazepin.Homework4/Program.cs (hunk 1 above)
Comment (Mazepin) [reply src/Mazepin.Homework4/Program.cs]: I know ) Still in progress

**Comment by Mazepin** [comment]
Check if you have time. Added examples of inheritance and polymorphism in the application

**Pull request merged**